distributions/openhab/src/main/resources/conf/services/addons.cfg

```diff
@@ -3,18 +3,13 @@
 # Note: If you want to specify your add-ons yourself through entries below, set the package to "minimal"
 # as otherwise your definition might be in conflict with what the installation package defines.
 #
-# Optional. If not set, the dashboard (https://<yourserver>:8080/) will ask you to choose a package.
-#
 # Valid options:
 #   - minimal  : Installation only with dashboard, but no UIs or other add-ons. Use this for custom setups.
-#   - simple   : Setup for using openHAB purely through UIs - you need to expect MANY constraints in functionality!
 #   - standard : Default setup for normal users, best for textual setup
-#   - expert   : Setup for expert users, especially for people migrating from openHAB 1.x
-#   - demo     : A demo setup which includes UIs, a few bindings, config files etc.
 #
 # See https://www.openhab.org/docs/configuration/packages.html for a detailed explanation of these packages.
 #
-#package = minimal
+package = standard
```

Comment:
What would happen to the demo setup? It's sometimes useful.
The setup wizard could have a step asking users if they want "a few things configured for them" (that would install the standard package), a demo setup (install the demo package), or "skip" the step entirely (stay on minimal), wdyt?

Reply:
As written above: I would suggest to completely remove the package selection from the wizard. I for now kept the package mechanism still in place, so that "minimal" and "demo" can be picked for very special setups (e.g. our demo server). But the package is installed upon the very first startup already, so once the UI is there, there is no need to select anything anymore - it simply means the user has chosen a "standard" setup. Does that make sense to you?

Reply:
Yes it's clearer, I thought the minimal package included the UI (doesn't it though, because it's a boot feature rather than an add-on?)

Reply:
You're right, that's likely something we will have to change - adding the dashboard as a boot feature was only meant as a quick hack to solve the offline installation issues.